WooCommerce/Classes/ViewRelated/JetpackSetup/JetpackSetupCoordinator.swift

```diff
@@ -376,15 +376,16 @@ private extension JetpackSetupCoordinator {
 
     func show2FALoginUI(with loginFields: LoginFields) {
         analytics.track(event: .JetpackSetup.loginFlow(step: .verificationCode))
+        let window = rootViewController.view.window!
```

Comment:
Instead of force unwrapping `window`, would initializing a new window [like this](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/trunk/WordPressAuthenticator/Signin/AppleAuthenticator.swift#L206) as a fall back work? If not, we can use `logErrorAndExit` to crash and log an error.

Reply:
Good idea! Updated the code to use `logErrorAndExit` in e1faa9f8998437bd0841f43d31ba076474093a87.